README.adoc

```diff
@@ -64,6 +64,7 @@ For more information about enabling, disabling, and applying add-ons, see the li
 == CI Setup
 
 `minishift-addons` uses link:https://ci.centos.org/[CentOS CI] as CI build server.
+For a new pull request the link:https://ci.centos.org/job/minishift-addons-pr/[pull request build job] will be triggered. However, the build job runs the integration tests only when the pull request contains any change related to add-on.
```

Comment:
s/`For a new pull request the pull request build job will be triggered`/`A pull request build job will be triggered for every pull requests`/